cla_public/static-src/stylesheets/_notice.scss

```diff
@@ -1,9 +1,6 @@
+.get-in-touch,
 .notice {
-  border-top: 2px solid $grey-3;
```

Comment:
Was deleting this line intentional?

Reply:
Yes.  the border is now added in elsewhere (in the right colour).